examples/animation/pause_resume.py

```diff
@@ -0,0 +1,47 @@
+"""
+==================================
+Pausing and Resuming and Animation
+==================================
+
+This example showscases:
+- using the Animation.pause() method to pause an animation.
+- using the Animation.resume() method to resume an animation.
+"""
+
+import matplotlib.pyplot as plt
+import matplotlib.animation as animation
+import numpy as np
+
+
+class PauseAnimation:
+    def __init__(self):
+        fig, ax = plt.subplots()
+        ax.title.set_text('Click to pause/resume the animation')
```

Comment:
Can you replace with `ax.set_title('Click to pause/resume the animation')`?

Reply:
Done.